docs/guides/ai-transpiler-passes.mdx

```diff
@@ -102,6 +112,38 @@ The following custom collection passes for Cliffords, Linear Functions and Permu
 
 These custom collection passes limit the sizes of the collected sub-circuits so they are supported by the AI-powered synthesis passes. Therefore, it is recommended to use them after the routing passes and before the synthesis passes for a better overall optimization.
 
+## Hybrid Qiskit-AI circuit transpilation
+
+The `qiskit-ibm-transpiler` offers the option to configure a hybrid pass manager that combines the best of Qiskit and the AI-powered transpiler passes, so you can run a whole transpilation on your own. This feature behaves similarly to the Qiskit `generate_pass_manager` method. A typical way to use `generate_ai_pass_manager` is as follows:
+
+```python
+    from qiskit_ibm_transpiler.ai import generate_ai_pass_manager
+    from qiskit.circuit.library import EfficientSU2
+    from qiskit_ibm_runtime import QiskitRuntimeService
+
+    backend = QiskitRuntimeService().backend("ibm_fez")
+    fez_coupling_map = backend.coupling_map
+
+    su2_circuit = EfficientSU2(101, entanglement="circular", reps=1).decompose()
+
+    ai_transpiler_pass_manager = generate_ai_pass_manager(,
+        coupling_map=fez_coupling_map,
+        ai_optimization_level=3,
+        optimization_level=3,
+        ai_layout_mode=optimize,
+    )
+
+    ai_su2_transpiled_circuit = ai_transpiler_pass_manager.run(su2_circuit),
+```
+
+The following options are used:
+* `coupling_map` - Specifies which coupling map to use for the transpilation. 
+* `ai_optimization_level` - Specifies the level of optimization (1-3) to use for the AI components of the PassManager.
+* `optimization_level` - Specifies the optimization to use for the Qiskit heuristic algorithms. 
+* `ai_layout_mode` - Specifies how the AI routing part of the PassManager handles the layout. 
+
+Refer to the #AI-routing-pass section to review the configuration options for this parameter.
```

Comment:
What parameter?

Reply:
It has a line break that shouldn't be there. Fixed in https://github.com/Qiskit/documentation/pull/2273/commits/aa8575f5eaad17a6bbe6b6a913fb3f563c925ee0